Lib/asyncio/base_events.py

```diff
@@ -179,6 +179,9 @@ def close(self):
         if self._active_count == 0:
             self._wakeup()
 
+    def get_loop(self):
+        return self._loop
```

Comment:
Please add a test for the method.

Reply:
I cannot find tests for Server  class. Could you help me please?

Reply:
`loop.create_server` returns Server objects.